modules/xmpp/RtxModifier.js

```diff
@@ -163,62 +110,64 @@ export default class RtxModifier {
      * @param {string} sdpStr sdp in raw string format
      */
     modifyRtxSsrcs (sdpStr) {
-        let parsedSdp = transform.parse(sdpStr);
-        let videoMLine = 
-            parsedSdp.media.find(mLine => mLine.type === "video");
-        if (videoMLine.direction === "inactive" ||
-                videoMLine.direction === "recvonly") {
+        const sdpTransformer = new SdpTransformWrap(sdpStr);
+        if (!sdpTransformer.selectMedia("video")) {
+            logger.error("No 'video' media found in the sdp: " + sdpStr);
+            return sdpStr;
+        }
+        const direction = sdpTransformer.mediaDirection;
+        if (direction === "inactive" || direction === "recvonly") {
             logger.debug("RtxModifier doing nothing, video " +
                 "m line is inactive or recvonly");
             return sdpStr;
         }
-        if (!videoMLine.ssrcs) {
+        if (sdpTransformer.getSSRCCount() < 1) {
```

Comment:
I'm just trying to handle all cases even those that are "impossible". What if for any reason getSSRCCount would return -1 ? The code would try to execute with some possibly unexpected behaviour or crash.